apps/web/vercel.json

```diff
@@ -23,6 +23,9 @@
     },
     "pages/api/trpc/appRoutingForms/[trpc].ts": {
       "memory": 2048
+    },
+    "pages/router/embed.tsx": {
```

Comment:
Not sure if page route will work but I am assuming it would.

Giving it more vCPUs, because there are a lot of computations that happen

Reply:
We could add non embed route as well later but for now this is necessary.